dateparser/conf.py

```diff
@@ -193,6 +206,13 @@ def check_settings(settings):
         'PREFER_LOCALE_DATE_ORDER': {
             'type': bool
         },
+        'DEFAULT_LANGUAGES': {
+            'type': list
+        },
```

Comment:
I think we could add an `extra_check` that checks that the values in `DEFAULT_LANGUAGES` are valid.

This is not strictly necessary, as now, when we try with a wrong language it raises an error:

```python
>>> dateparser.parse('20/20/2020', settings={'DEFAULT_LANGUAGES': ['aaa']})
ValueError: Unknown language(s): 'aaa'
```

but for example, when using `DateDataParser`, the error only appears when trying to get the date, and adding this check here would show the error when instantiating, which seems better.

```python
>>> dd = DateDataParser(settings={'DEFAULT_LANGUAGES': ['aaa']})
>>> dd.get_date_data('12/12/2000')
ValueError: Unknown language(s): 'aaa'
```

I think the way to implement this new check could be exactly the same that is producing that error. After doing this we should add a test to check this (similar to what we did for the `LANGUAGE_DETECTION_CONFIDENCE_TRESHOLD`.

Anyways, feel free to open a new / separate issue to handle this.

Reply:
Let this conversation open until we clarify what I put about this setting in my comments below.

Reply:
I have created some test but I am certain its wrong please suggest